integration/setup/setup.go

```diff
@@ -32,12 +38,6 @@ func WrapTestMain(c Config, m *testing.M) {
 		v = m.Run()
 	}
 
-	err = teardown(c)
```

Comment:
For my curiosity: do we do any teardown at all or we decided we abandon the cluster after the test so we don't need it?

Reply:
I think circleCI would tear down a KIND cluster anyway :)

Reply:
Yes for these app tests there are no external resources.

Circle clean up their VMs and when testing locally I usually delete the KIND cluster to have a clean starting state.